native/src/modules/endpoint/reducers/pushCategory.js

```diff
@@ -46,6 +46,12 @@ const pushCategory = (state: CityContentStateType, action: PushCategoryActionTyp
     }
   }
 
+  // Check whether another page in the same city is loading, e.g. because it is being refreshed.
+  // This is important for displaying the loading spinner.
+  const otherPageLoading = Object.values(state.categoriesRouteMapping)
+    .filter(route => city === route.city && path !== route.path)
+    .reduce((acc, current) => (acc || current.status === 'loading'), false)
```

Comment:
Why not use `.some()`?
```suggestion
    .some(route => current.status === 'loading')
```

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
Didn't know this existed :smile: